training/pytorch/structured/custom_containers/base/README.md

```diff
@@ -0,0 +1,75 @@
+# PyTorch Custom Containers Template
```

Comment:
Update versions below to show Pytorch instead of TensorFlow?

Reply:
Whoops, updated